queries.yaml

```diff
@@ -273,3 +273,28 @@ post_process:
     params:
       source_layer: roads
       start_zoom: 9
+  - fn: TileStache.Goodies.VecTiles.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 14
+      properties: [is_bridge, is_tunnel, oneway]
```

Comment:
I'm eager to start trying it out!! :)

On Thu, Jan 7, 2016 at 11:22 AM, Nathaniel V. KELSO <
notifications@github.com> wrote:

> In queries.yaml
> <https://github.com/mapzen/vector-datasource/pull/468#discussion_r49091365>
> :
>
> > @@ -273,3 +273,28 @@ post_process:
> >      params:
> >        source_layer: roads
> >        start_zoom: 9
> > +  - fn: TileStache.Goodies.VecTiles.transform.drop_properties
> > +    params:
> > +      source_layer: roads
> > +      start_zoom: 0
> > +      end_zoom: 14
> > +      properties: [is_bridge, is_tunnel, oneway]
>
> Should we put this on dev (without addressing this) so we can compare
> with prod and then revise if necessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mapzen/vector-datasource/pull/468/files#r49091365>.
>